terraform/alerting/monitoring.tf

```diff
@@ -27,3 +27,49 @@ resource "google_monitoring_dashboard" "e2e" {
     null_resource.manual-step-to-enable-workspace
   ]
 }
+
+resource "google_monitoring_alert_policy" "five_xx" {
+  project      = var.project
+  display_name = "Elevated 5xx"
+  combiner     = "OR"
+  conditions {
+    display_name = "Elevated 5xx on Verification Server"
+    condition_threshold {
+      duration        = "300s"
+      threshold_value = 2
+      comparison      = "COMPARISON_GT"
+      filter          = "metric.type=\"run.googleapis.com/request_count\" resource.type=\"cloud_run_revision\" metric.label.\"response_code_class\"=\"5xx\" resource.label.\"service_name\"!=\"e2e-runner\""
```

Comment:
Should we also define these as YAML files?

Reply:
Hmm, I guess we could? hadn't thought about it. How about I do in a follow up PR?

Reply:
SG